modules/schematics/src/entity/schema.json

```diff
@@ -49,6 +49,11 @@
       "description":
         "Group actions, reducers and effects within relative subfolders",
       "aliases": ["g"]
+    },
+    "plural": {
```

Comment:
This isn't needed, we always pluralize - https://github.com/ngrx/platform/blob/757d04ebeb35a8811c214a3b3c9e02318e650034/modules/schematics/src/entity/index.ts#L34

Reply:
removed